node/runtime/src/lib.rs

```diff
@@ -362,6 +364,7 @@ impl Runtime {
                         self.staking(
                             state_update,
                             &t,
+                            &transaction.body.get_sender(),
```

Comment:
Probably better to refactor with the call to `transaction.body.get_sender()` at the beginning of this function.